uninstall.php

```diff
@@ -32,8 +32,6 @@
 		'edac_frontend_highlighter_position',
 		'edac_delete_data',
 		'edac_review_notice',
-		'edac_authorization_password',
-		'edac_authorization_username',
```

Comment:
We should still keep these here so they get removed on plugin uninstall for anyone that had the plugin installed before we removed this feature, maybe with a commend noting they are legacy.